code/game/objects/structures/crates_lockers/closets/secure/bar.dm

```diff
@@ -9,3 +9,4 @@
 	..()
 	for(var/i in 1 to 10)
 		new /obj/item/reagent_containers/food/drinks/beer( src )
+		new /obj/item/etherealballdeployer(src)

```

Comment:
It will contain 10 beers, AND 10 ethereal balls. Are you sure this is what you meant

Reply:
feature